---
title: "Welch Lab - Software"
layout: piclay
excerpt: "Welch Lab -- Software"
permalink: /software
---

# Software

**LIGER (Linked Inference of Genomic Experimental Relationships)**

[https://github.com/welch-lab/liger](https://github.com/welch-lab/liger)

**PyLiger: Scalable single-cell multi-omic data integration in Python**

[https://github.com/welch-lab/pyliger](https://github.com/welch-lab/pyliger)

**MultiVelo: Single-cell multi-omic velocity inference**

[https://github.com/welch-lab/MultiVelo](https://github.com/welch-lab/MultiVelo)

**VeloVAE: Bayesian Inference of RNA Velocity from Multi-Lineage Single-Cell Data**

[https://github.com/welch-lab/VeloVAE](https://github.com/welch-lab/VeloVAE)

**PerturbNet: predicting single-cell responses to perturbations**

[https://github.com/welch-lab/PerturbNet](https://github.com/welch-lab/PerturbNet)

**MorphNet: predicting cell morphology from single-cell gene expression**

[https://github.com/welch-lab/MorphNet](https://github.com/single-cell-morphology)

**MichiGAN (Sampling from Disentangled Representations of Cell Identity using Generative Adversarial Networks)**

[https://github.com/welch-lab/MichiGAN](https://github.com/welch-lab/MichiGAN)

**SquiggleNet (Real-Time, Direct Classification of Nanopore Signals)**

[https://github.com/welch-lab/SquiggleNet](https://github.com/welch-lab/SquiggleNet)

**MATCHER (Manifold Alignment to Characterize Experimental Relationships)**

[https://github.com/jw156605/MATCHER](https://github.com/jw156605/MATCHER)

**SLICER (Selective Locally Linear Inference of Cellular Expression Relationships)**

[https://github.com/jw156605/SLICER](https://github.com/jw156605/SLICER)

**SingleSplice algorithm for detecting alternative splicing in a population of single cells**

[https://github.com/jw156605/SingleSplice](https://github.com/jw156605/SingleSplice)

**AppEnD tool for mapping 3â€™ additions from RNA-seq data**

[https://github.com/jw156605/append](https://github.com/jw156605/append)
