---
title: "Welch Lab - Research"
layout: textlay
excerpt: "Welch Lab -- Research"
sitemap: false
permalink: /research/
---

# Research Summary

Our research aims to address fundamental problems in both biomedical research and computer science by developing new tools tailored to rapidly emerging high-throughput sequencing technologies. Broadly, we seek to understand what genes define the complement of cell types and cell states within healthy tissue, how cells differentiate to their final fates, and how dysregulation of genes within specific cell types contributes to human disease. As computational method developers, we seek to both employ and advance the methods of machine learning, particularly for unsupervised analysis of high-dimensional data.

Most recently, I have focused on developing open-source software for the processing, analysis, and modeling of single-cell sequencing data. Key contributions in this area include SLICER, an algorithm for inferring developmental trajectories; LIGER, a general approach for integrating single-cell transcriptomic, epigenomic and spatial transcriptomic data; and online iNMF, a scalable and iterative algorithm for single-cell data integration. I have applied these methods in collaboration with biological scientists to study stem cell differentiation, somatic cell reprogramming, and the brain.

**Single-Cell Data Integration**.
![]({{ site.url }}{{ site.baseurl }}/images/slider7001400/online-inmf.png){: style="width: 250px; float: left; margin: 0px  10px"}
In recent papers published in Cell and Nature Biotechnology, we developed LIGER and online iNMF. These approaches allow integration of single-cell transcriptomic, epigenomic and spatial transcriptomic datasets from different single cells.

**Single-Cell Trajectory Inference**.
![]({{ site.url }}{{ site.baseurl }}/images/respic/research2.png){: style="width: 250px; float: left; margin: 0px  10px"}
We developed SLICER and MATCHER, approaches for inferring sequential transcriptomic and epigenomic changes in differentiating cells.

**Applications in Neuroscience**.
![]({{ site.url }}{{ site.baseurl }}/images/slider7001400/atlas.png){: style="width: 250px; float: left; margin: 0px  10px"}
As part of the BRAIN Initiative Cell Census Network, we are integrating single-cell transcriptomic, epigenomic, and spatial datasets to characterize the cell types in the mouse brain.

**Applications in Stem Cell Differentiation and Reprogramming**.
![]({{ site.url }}{{ site.baseurl }}/images/slider7001400/icm.png){: style="width: 250px; float: left; margin: 0px  10px"}
Using the algorithms we have developed, we are investigating stem cell differentiation in mouse bone and human blood cells. We are also interested in somatic cell reprogramming, particularly direct cardiac reprogramming.

![]({{ site.url }}{{ site.baseurl }}/images/slider7001400/liger-cell.jpg){: style="width: 70%; float: center; margin: 0px"}

